### PR TITLE
Gdb 3694 replace static libraries

### PR DESCRIPTION
**Pull request by svilenvelikov**
* Replace `lodash` with latest version from npm.
* Replace `jquery` with latest version from npm.
  * There are changes in $.param method which result different outcome.
  * There are changes in $.ajax method callback functions names (e.g. success -> done).
  * Add new tests and update existing to account changes in jquery.
  * Reorganized cypress tests for similarity index because of an issue with cypress loosing the repository cookie after a second cy.visit.
* Remove `requirejs`.
  * It isn't used anymore.
* Replace `angular-xeditable` with latest version from npm.
  * Copy lib stylesheet from node_modules to dist and update references.
* Replace `angular-pageslide-directive` with latest version from npm.
  * Refactored places where this directive is used to prevent errors caused by onopen and onclose functions present in angular scope which deludes the library that those are passed by the client and because of their wrong outcome it throws an error.  
* Replace `angular-bowser` with latest version from npm.
* Replace `autofill-event` with latest version from npm.
* Replace `angularjs-slider` with latest version from npm.
* Replace `ng-file-upload` and the ng-file-upload shim with latest version from npm.
* Replace `angular-toastr` with latest version from npm.
* Replace `bootstrap-switch` with latest version from npm.
* Replace `angular-ui-scroll` with latest version from npm.
* Replace `ng-tags-input` with latest version from npm.
* Replace `font-awesome` with the latest 4-th version because the 5-th version has changes in the license and packaging.
* Replace `d3.js` with concrete version 4.2.1 from npm.
* Delete the `stringify` library because it's not used.
* Move `loading-hint` in application's source folder because it's not a library.
* Replace `angular-animate` with concrete version 1.3.8 from npm.
* Replace `angular-route` with concrete version 1.3.8 from npm.
* Replace `angular-local-storage` with concrete version 0.1.5 from npm.
* Replace `angular-sanitize` with concrete version 1.3.11 from npm.
* Replace `angular-ui-bootstrap` with concrete version 0.13.0 from npm.

Additional fixes:
* Put variable identifiers in front of `msg` variable to prevent errors.
* Removed the proxy definition form package.json.

Documentation for library issues which are found https://docs.google.com/document/d/1NwZHYBkQ6EfszZ5eTFBBa9L4P0Cf6TFTnmuODs03Ta0/edit#heading=h.r7xngoml3ga7
Jira: https://ontotext.atlassian.net/browse/GDB-3694

### CODE REVIEW

File: test-cypress/integration/sparql/sparql.menu.spec.js [hunk 1]
```diff
@@ -732,7 +732,7 @@ describe('SPARQL screen validation', () => {
 
             // TODO: Test with different values for infer & same as
 
-            const encodedQuery = 'SELECT+%3Fsub+%3Fpred+%3Fobj+WHERE+%7B%3Fsub+%3Fpred+%3Fobj+.%7D+LIMIT+100';
+            const encodedQuery = 'SELECT%20%3Fsub%20%3Fpred%20%3Fobj%20WHERE%20%7B%3Fsub%20%3Fpred%20%3Fobj%20.%7D%20LIMIT%20100';
```
Comment (svilenvelikov) [inline test-cypress/integration/sparql/sparql.menu.spec.js]: jquery $.param in newer versions escapes it this way

File: package.json [hunk 2]
```diff
@@ -69,5 +69,9 @@
         "webpack-dev-server": "^3.7.2",
         "webpack-merge": "^4.2.1"
     },
-    "proxy": "http://localhost:7200"
+    "proxy": "http://localhost:7200",
```
Comment (mihailradkov) [inline package.json]: Hm, I though that we had removed the proxy.

Comment (svilenvelikov) [reply package.json]: No we don't.

**Comment by svilenvelikov** [comment]
This is a WIP PR.

**Pull request merged**